src/diffpy/utils/parsers/__init__.py

```diff
@@ -17,11 +17,10 @@
 """
 
 from .loaddata import loadData
-from .resample import resample
 from .serialization import deserialize_data, serialize_data
 
 # silence the pyflakes syntax checker
-assert loadData or resample or True
+assert loadData or True
```

Comment:
I am afraid this isn't the latest cookiecutting standard for handling `__init__.py` files within `src`. Could you please check whether we have to import these functions lilke `from .loaddata import loadData`?

Reply:
No we don't, should I delete these then?